templates/nextflow-ecs-cluster.j2

```diff
@@ -135,17 +107,35 @@ Resources:
       AssociatePublicIpAddress: true
       IamInstanceProfile: !Ref EcsInstanceProfile
       SecurityGroups:
-        - !Ref EcsSecurityGroup
+        - !Ref EcsSecurityGroupId
       BlockDeviceMappings:
         - DeviceName: !Ref RootDeviceName
           Ebs:
             VolumeSize: !Ref RootEbsVolumeSize
             VolumeType: !Ref EbsVolumeType
       UserData:
         Fn::Base64: !Sub |
-          #!/bin/bash
-          echo ECS_CLUSTER=${EcsClusterName} >> /etc/ecs/ecs.config
-          echo ECS_BACKEND_HOST= >> /etc/ecs/ecs.config
+          MIME-Version: 1.0
+          Content-Type: multipart/mixed; boundary="==BOUNDARY=="
+
+          --==BOUNDARY==
+          Content-Type: text/cloud-config; charset="us-ascii"
+
+          #cloud-config
+          repo_update: true
+          repo_upgrade: security
+
+          write_files:
+          - path: /opt/nextflow/tower.yml
+            content: |
+              {{ sceptre_user_data.TowerConfigFileContents|indent(14) }}
+            permissions: '0755'
+          - path: /etc/ecs/ecs.config
+            content: |
+              ECS_CLUSTER=${EcsClusterName}
+              ECS_BACKEND_HOST=
```

Comment:
This is commonly seen in ecs.config files; it's a placeholder if you want to set the ecs endpoint. It's not necessary and I will remove to avoid confusion.